### PR TITLE
add [pt-br] /content/pt-br/docs/concepts/worklouds/controllers/ttlafterfinished.md

### PR DESCRIPTION
**Pull request by gpersijn**

### CODE REVIEW

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+reviewers:
```
Comment (cpanato) [inline content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:2]: do we need to have this?

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+title: Automatic Clean-up for Finished Jobs
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} provides a
```
Comment (cpanato) [inline content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:13]: missing translation here

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Os segundos TTL podem ser definidos a qualquer momento. Aqui estão alguns exemplos para configurar o
+Campo `.spec.ttlSecondsAfterFinished` de um Trabalho:
+
+- Especifique este campo no manifesto do trabalho, para que um trabalho possa ser limpo automaticamente algum tempo depois de terminar.
+- Defina este campo de trabalhos existentes, já finalizados, para adotar esta nova funcionalidade.
+- Utilize a [mutating admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks) para configurar esse campo dinamicamente no tempo de criação do Trabalho. Os administradores de cluster podem usar isso para impor uma política TTL para trabalhos concluídos.
+- Utilize a [mutating admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks) para configurar este campo dinamicamente depois do trabalho ter sido concluído, e escolha diferentes valores TTL baseados em status do trabalho, rótulos, etc.
+
+## Embargo
```
Comment (cpanato) [inline content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:41]: maybe `aviso` or `advertência` fits better here

File: content/pt-br/docs/concepts/workloads/pods/ephemeral-containers.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<!-- overview -->
+
+{{< feature-state state="beta" for_k8s_version="v1.23" >}}
+
+Esta página fornece uma visão geral dos contêineres efêmeros: um tipo especial de contêiner que é executado temporariamente em um existente {{< glossary_tooltip term_id="pod" >}} para realizar ações iniciadas pelo usuário, como solução de problemas. Você usa
+contêineres efêmeros para inspecionar serviços em vez de construir aplicativos.
+
+<!-- body -->
+
+## Entendendo containeres efêmeros
```
Comment (cpanato) [inline content/pt-br/docs/concepts/workloads/pods/ephemeral-containers.md:19]:
```suggestion
## Entendendo contêineres efêmeros
```

File: content/pt-br/docs/concepts/workloads/pods/ephemeral-containers.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+- Contêineres efêmeros podem não ter portas, então campos como `ports`, `livenessProbe`, `readinessProbe` não são permitidos.
+- As alocações de recursos do pod são imutáveis, portanto, a configuração de `resources` não é permitida.
+- Para obter uma lista completa de campos permitidos, consulte a [referência de EphemeralContainer documentação](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#efemeralcontainer-v1-core).
+
+Os contêineres efêmeros são criados usando um manipulador especial `ephemeralcontainers` na API em vez de adicioná-los diretamente ao `pod.spec`, então não é possível adicionar um contêiner efêmero usando `kubectl edit`.
+
+Assim como os contêineres comuns, você não pode alterar ou remover um contêiner efêmero depois de adicioná-lo a um pod.
+
+## Usos de containeres efêmeros
```
Comment (cpanato) [inline content/pt-br/docs/concepts/workloads/pods/ephemeral-containers.md:40]:
```suggestion
## Usos de contêineres efêmeros
```

File: content/pt-br/docs/concepts/workloads/pods/ephemeral-containers.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+## Entendendo containeres efêmeros
+
+{{< glossary_tooltip text="Pods" term_id="pod" >}} são as bases de construção de aplicativos Kubernetes. Como os Pods se destinam a ser descartável ​​e substituível, não é possível adicionar um contêiner a um pod depois de criá-lo.
+Em vez disso, você geralmente exclui e substitui Pods de maneira controlada usando {{< glossary_tooltip text="deployments" term_id="deployment" >}}.
+
+Às vezes, é necessário inspecionar o estado de um pod existente, no entanto, para
+exemplo para solucionar um bug difícil de reproduzir. Nesses casos, você pode executar um contêiner efêmero em um pod existente para inspecionar seu estado e executar
+comandos arbitrários.
+
+### O que é um container efêmero?
```
Comment (cpanato) [inline content/pt-br/docs/concepts/workloads/pods/ephemeral-containers.md:28]:
```suggestion
### O que é um contêiner efêmero?
```

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} provides a
+TTL (time to live) mechanism to limit the lifetime of resource objects that
+have finished execution. TTL controller only handles
+{{< glossary_tooltip text="Jobs" term_id="job" >}}.
+
+<!-- body -->
+
+## Controlador TTL-after-finished
+
+O controlador TTL-after-finished é suportado apenas para Trabalhos. Um operador de cluster pode usar esse recurso para limpar
```
Comment (cpanato) [inline content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:22]: are we using the literal translation of `jobs` to `Trabalhos`? should we keep `job` and not translate that at all?

**Review by cpanato — CHANGES_REQUESTED** [review]
thanks for working on this
some comments

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md (hunk 7 above)
Comment (stormqueen1990) [reply content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:22]: We have standardized the translation of "job" as "tarefa" in Brazilian Portuguese.

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md (hunk 1 above)
Comment (stormqueen1990) [reply content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:2]: It's OK to remove reviewers.

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
+---
+reviewers:
+  - janetkuo
+title: Automatic Clean-up for Finished Jobs
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:4]: Missing translation for title.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @gpersijn, thanks for your contribution! 🎉 

I noticed you added two different files to the same PR (controllers/ttlafterfinished and pods/ephemeral-containers). Could you please split those into two separate PRs?

File: content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md (hunk 7 above)
Comment (cpanato) [reply content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md:22]: perfect sounds great

**Comment by cpanato** [comment]
> Hi @gpersijn, thanks for your contribution! 🎉
> 
> I noticed you added two different files to the same PR (controllers/ttlafterfinished and pods/ephemeral-containers). Could you please split those into two separate PRs?

sounds like a good idea to keep the PR small and easier to review, is it possible to split it @gpersijn when you have some free cycles? thanks again for working on this

**Comment by MrErlison** [comment]
Hi @gpersijn 

Your contribution is important to us Portuguese speakers and is well on its way. 

We are here to help you and open to answer any questions, criticism or suggestions. To do so, just access our channel #kubernetes-doc-pt on slack that is always available for you. 

We are waiting for your return.

---

Sua contribuição é importante para nós falantes da língua portuguesa e está bem encaminhada. 

Estamos aqui para ajudá-lo e abertos para sanar qualquer dúvida, crítica ou sugestão. Para isso basta acessar o nosso canal no slack #kubernetes-doc-pt que está sempre disponível para recebê-lo.

Estamos aguardando o seu retorno.

**Comment by stormqueen1990** [comment]
Hello there, @gpersijn! :wave: 

Thank you very much for your work on this PR. However, it has been open and reviewed for a while with no action taken. For this reason, I will be closing this PR. If you wish to continue your work on this, please feel free to reopen it.

Should you have any questions, please don't hesitate to reach out to the `#kubernetes-docs-pt` channel in Kubernetes Slack.

---

Olá, @gpersijn! :wave:

Obrigada pelo teu trabalho neste PR. Devido a inatividade após a revisão, irei fechá-lo. No entanto, caso desejes continuar teu trabalho, podes reabrir o PR a qualquer momento.

Se tiveres dúvidas, não hesita em contatar-nos através do canal `#kubernetes-docs-pt` no Slack do Kubernetes.

/close

**Pull request closed without merging**